maintenance/maintenance.json

```
@@ -0,0 +1,4 @@
+{
+  "show": false,
+  "message": "Maintenance scheduled for tomorrow at 13:00"
```

```suggestion
  "message": "Sorry, we are down for scheduled maintenance."
```
From https://github.com/ral-facilities/scigateway-auth/blob/main/maintenance/maintenance.json, this is what is displayed when the system is "down".